src/fu-tool.c

```diff
@@ -1303,7 +1303,7 @@ fu_util_install_release(FuUtilPrivate *priv, FwupdRelease *rel, GError **error)
 }
 
 static gboolean
-fu_util_update_all(FuUtilPrivate *priv, GError **error)
+fu_util_update_all(FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
similar to `fu-util.c`:
since you essentially combined `fu_util_update_all` and `fu_util_update_by_id` into a single function suggest this function should just move into `fu_util_update`